### ISSUE TITLE
Add tag 'download' in case a song must be downloaded from CF

### ISSUE THREAD

**Issue by kozaka-tv** [kozaka-tv/Rocksmith-Servant#101]
From SongLoader::run()

After: if time_waited >= HEARTBEAT:

TODO download dir(s). !!! It could be more than one!
self.__update_songs_from_download_dir()
self.__update_songs_from_import_dir()
TODO tmp dir?!
self.__update_songs_from_tmp_dir()

**Comment by kozaka-tv** [comment]
This function is already done!
